app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/FlankerViewModel.java

```diff
@@ -102,4 +121,18 @@ public String arrowText() {
     boolean showLeft = rand.nextBoolean(); // TODO - not random.
     return showLeft ? ">><>>" : ">>>>>";
   }
+
+  /** Update after user tap, return true if we recorded it. */
+  public boolean handleScreenTap(boolean isOnLeft) {
```

Comment:
For any event system (android, ios, web, ...), the usual thing is to return a boolean:
True if you dealt with the event, false otherwise; in the case of false, it'll bubble upwards, which is why we get tap events on the root view, even if you tap on a TextView within it. 

Which is why here we return true only when storing the tap.